portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Nikita!");
-  }
+    ArrayList<String> comments; 
+
+    public DataServlet() {
+        this.comments = new ArrayList<String>();
```

Comment:
why does this need to be set in the construct? why not make this variable local to the doGet method.

Reply:
changed - might have done it originally in anticipation for dynamic comments